src/miner.cpp

```diff
@@ -99,6 +99,9 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
     // Limit to between 1K and (hard limit - 1K) for sanity:
     nBlockMaxSize = std::max((uint64_t)1000, std::min((hardLimit - 1000),  nBlockMaxSize));
 
+    // For compatibility with bip68-sequence test, set flag to not mine txs with negative fee delta.
+    const bool fSkipNegativeDelta = GetBoolArg("-bip68hack", false);
```

Comment:
> It's possible a miner may want to relay certain transactions but not mine them himself. Any interest in naming this -minenegativedelta (even defaulting to false), ...

I'd like for a miner to present me with a use case where this is desirable, then I'd be happy to rename it. For now, I think it's better to keep the current name to emphasize it's a feature we want to remove.

> ... or removing the option completely, since our usage of fee delta is already unique to XT?

I'd like that, but the bip68 rpc test needs to change then.